src/test/4_FloorERC721PricingModule.t.sol

```diff
@@ -152,7 +152,7 @@ contract FloorERC721PricingModuleTest is Test {
         vm.stopPrank();
     }
 
-    function testNonOwnerAddsAsset(address unprivilegedAddress) public {
+    function testRevert_AssetInformation_NonOwnerAddsAsset(address unprivilegedAddress) public {
```

Comment:
testRevert_setAssetInformation_NonOwnerAddsAsset

Also for next functions don't forget the set